.github/pull_request_template.md

```diff
@@ -1,16 +1,23 @@
+## Description
+
+1. What is this PR about (link the issue and add a short description)
+
+## Steps for reproduction
+
+1. click button
+2. expect xyz
```

Comment:
I want us to copy the steps from the issue if they are already there, it makes it easier to review. For pr creator its 10 seconds of work for someone who reads it can be confusing if issue is complex.

Reply:
Basically don't rely on the issue. Describe the PR. Issue can be very convoluted due to all conversations.

Reply:
Still link the issue though, just don't expect reviewer to read the entire issue.